### DISCUSSION TITLE
Can I get an invite to the Discord Emoji Server?

### DISCUSSION THREAD

**Discussion by priyam-raj** [Androz2091/AtlantaBot#470] [Q&A] [answered]
The current link does not work ("https://www.atlanta-bot.fr/emojis"), please fix the invite link to a permanent one.

**Comment by Androz2091** [comment, marked as answer]
**https://emojis.atlanta-bot.fr/**

**Reply by priyam-raj** [reply]
Emoji server, not these ones?

**Reply by priyam-raj** [reply]
![image](https://user-images.githubusercontent.com/26673536/117872938-e164af80-b2bc-11eb-9128-077286bf7681.png)
![image](https://user-images.githubusercontent.com/26673536/117872956-e45fa000-b2bc-11eb-9f06-cac9f82d27e5.png)
This one "https://www.atlanta-bot.fr/emojis"

**Reply by priyam-raj** [reply]
> https://androz2091.fr/discord or https://discord.gg/QgPes2wQ

Help.
